src/Providers/Address/OSPlacesAddressProvider.cs

```diff
@@ -70,24 +36,19 @@ public async Task<IEnumerable<AddressSearchResult>> SearchAsync(string streetOrP
             try
             {
                 if (streetOrPostcode.Contains(":full"))
-                {
                     response = await _gateway.GetAsync($"{_oSPlacesAddressProviderConfiguration.Host}?postcode={postcode}&fq=CLASSIFICATION_CODE:R*%20CLASSIFICATION_CODE:R*%20CLASSIFICATION_CODE:C*&key={_oSPlacesAddressProviderConfiguration.Key}&dataset=LPI");
-                }
                 else
-                {
                     response = await _gateway.GetAsync($"{_oSPlacesAddressProviderConfiguration.Host}?postcode={postcode}&fq=local_custodian_code:{_oSPlacesAddressProviderConfiguration.LocalCustodianCode}&fq=CLASSIFICATION_CODE:R*%20CLASSIFICATION_CODE:R*%20CLASSIFICATION_CODE:C*&key={_oSPlacesAddressProviderConfiguration.Key}&dataset=LPI");
-                }
-                _logger.LogWarning($"OSPlaces Address provider:: response {response.StatusCode} {response.RequestMessage}");
             }
             catch (Exception ex)
             {
                 _logger.LogWarning($"OSPlaces Address provider:: exception thrown  {ex.Message}");
```

Comment:
If you're going to leave the log in it'd probably be better to log what you've got in the exception below